src/main/webapp/js/feed-loader.js

```diff
@@ -1,3 +1,21 @@
+
+
+/**
+ * Shows the message form if the user is logged in.
+ */
+ function showMessageFormIfLoggedIn() {
+   fetch('/login-status')
+    .then((response) => {
+      return response.json();
+    })
+    .then((loginStatus) => {
+      if (loginStatus.isLoggedIn) {
```

Comment:
Why are we removing the messageForm when user is logged in?

Reply:
The message form is set as hidden, and we will show the message form if the user is logged in. Here I check if the user is logged in and remove the class "hidden" if yes, not the messageForm.

Reply:
I see. That makes sense. Might be useful to include a comment here.